pkg/render/logstorage.go

```diff
@@ -383,8 +397,27 @@ func (es elasticsearchComponent) podTemplate() corev1.PodTemplateSpec {
 		}
 	}
 
+	// Was already present, but hidden from view in ECK 0.9. We are investigating how to improve on this internally:
```

Comment:
Since this is an opensource repo I'm not sure we should mention what we're doing internally...